tensorboard/plugins/projector/vz_projector/vz-projector-projections-panel.ts

```diff
@@ -168,6 +170,16 @@ export class ProjectionsPanel extends ProjectionsPanelPolymer {
     }
 
     this.runTsneButton.addEventListener('click', () => this.runTSNE());
+    this.pauseTsneButton.addEventListener('click', () => {
```

Comment:
How about changing the existing stop button to be "pause"/"resume" depending on whether t-sne is currently running. This way we can keep the UI cleaner and don't have to add another button.

Reply:
Great idea, the Stop button lends itself to some extra functionality. Come to think of it, there's no real need to explicitly terminate T-SNE (which is what Stop does), when one can re-run and pause. I'll split the pause-resume and sprite zoom functionalities and supersede with new pull requests.